Flights/Flight.cs

```diff
@@ -0,0 +1,180 @@
+﻿using Airport_Ticket_Booking_System.Utilites;
+namespace Airport_Ticket_Booking_System.Flights;
+
+public class Flight
+{
+    private int _flightNumber;
+    public int FlightNumber
+    { 
+        get 
+        { 
+            return _flightNumber;
+        } 
+        set 
+        {
+
+            if (value < 0)
+                throw new Exception("Flight Number must be positive");
+
+            _flightNumber = value;
+        }
+    }
+
+    private decimal _price;
+    public decimal Price 
+    {
+        get 
+        {
+            return _price;
+        }
+            
+        set
+        { 
+            if (value < 0) 
+                throw new Exception("Price must be positive");
+            _price = value;
+        }
+    }
+    private string _destination;
+    public string Destination { get { return _destination; } 
+        set 
+        {
+            if (string.IsNullOrEmpty(value))
+                throw new Exception("Destination cannot be empty");
+            if (value.Length < 3)
+                throw new Exception("Destination must be at least 3 characters long or more");
+            _destination = value;
+        } 
+    }
+    private string _departureAirport;
+    public string DepartureAirport { get { return _departureAirport; } 
+        set 
+        { 
+            if (string.IsNullOrEmpty(value))
+                throw new Exception("Departure Airport cannot be empty");
+            if (value.Length < 3)
+                throw new Exception("Departure Airport must be at least 3 characters long or more");   
+            _departureAirport = value;
+        } 
+    }
+    private string _arrivalAirport;
+    public string ArrivalAirport { get { return _arrivalAirport; } 
+        set
+        {
+            if (string.IsNullOrEmpty(value))
+                throw new Exception("Arrival Airport cannot be empty");
+            if (value.Length < 3)
+                throw new Exception("Arrival Airport must be at least 3 characters long or more");
+            _arrivalAirport = value;
+        } 
+    } 
+    private DateTime _departureDate;
+    public DateTime DepartureDate {
+        get
+        {
+            return _departureDate;
+        }
+        set
+        {
+            if (value < DateTime.Now)
+                throw new Exception("Departure Date must be in the future");
+            _departureDate = value;
+        }
+    }
+    public FlightClass Class { get; set; } = FlightClass.Economy;
+
+    public static string ToCsv(Flight flight)
+    {
+        return $"{flight.FlightNumber},{flight.Price},{flight.Destination},{flight.DepartureAirport},{flight.ArrivalAirport},{flight.DepartureDate},{flight.Class}";
+    }
+
+    public static Flight FromCsv(string csv, int line = 1)
+    {
+        string[] values = csv.Split(',');
+        List<string> errors = [];
+
+        Flight flight = new();
+
+        try
+            {
+            flight.FlightNumber = int.Parse(values[0]);
+        }
```

Comment:
Fix indentation. 
```suggestion
        try{
            flight.FlightNumber = int.Parse(values[0]);
        }
```

Apply this to all other try blocks.